jablib/src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2953,3 +2908,28 @@ You\ must\ select\ an\ identifier\ type.=You must select an identifier type.
 You\ must\ specify\ a\ Bib(La)TeX\ source.=You must specify a Bib(La)TeX source.
 You\ must\ specify\ an\ identifier.=You must specify an identifier.
 You\ must\ specify\ one\ (or\ more)\ citations.=You must specify one (or more) citations.
+
+# CommandLine
+Available\ export\ formats\:=Available export formats:
+Available\ import\ formats\:=Available import formats:
+Cannot\ embed\ metadata\ on\ any\ linked\ files\ of\ %s.\ Make\ sure\ there\ is\ at\ least\ one\ linked\ file\ and\ the\ path\ is\ correct.%n=Cannot embed metadata on any linked files of %s. Make sure there is at least one linked file and the path is correct.%n
+Cannot\ write\ XMP\ metadata\ on\ any\ linked\ files\ of\ %0.\ Make\ sure\ there\ is\ at\ least\ one\ linked\ file\ and\ the\ path\ is\ correct.%n=Cannot write XMP metadata on any linked files of %0. Make sure there is at least one linked file and the path is correct.%n
+Checking\ consistency\ of\ '%0'.=Checking consistency of '%0'.
+Checking\ integrity\ of\ '%0'.=Checking integrity of '%0'.
+Converting\ '%0'\ to\ '%1'=Converting '%0' to '%1'
```

Comment:
There is no final dot here, but the line above has.

Shouldn't we use three dots for an ongoing operation?